### PR TITLE
Added link to manage my booking

### LINKED ISSUE #147: Add link to manage my booking
**Acceptance criteria**
- When I click on "Manage my booking", I'm redirected to that page

screen shot 2018-05-07 at 16 00 31

Zeplin: https://zpl.io/adMXeDR

### PR DESCRIPTION
**Pull request by edgarordonez**
Close #147

Added:
 · Anchor tag to open in a new tab the URL of a booking.<br/><br/><br/><url>LiveURL: https://kiwicomsmart-faq-mnahvhqrzu.now.sh
</url>
</url>

### CODE REVIEW

File: src/SingleBookingPage/NearestBooking.js [hunk 1]
```diff
@@ -122,7 +122,13 @@ class NearestBooking extends React.Component<Props> {
           <Notification isUrgent={isUrgent} hoursLeft={deltaInHours} />
           {this.renderByType(nearestBooking.booking)}
           <div className="buttons">
-            <button className="manage-booking">Manage my booking</button>
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href={nearestBooking.booking.directAccessURL}
+            >
+              <button className="manage-booking">Manage my booking</button>
```
Comment (jalbertsr) [inline src/SingleBookingPage/NearestBooking.js]: what do you think about adding `cursor: pointer;` to the class? so it feels like is clickable

Comment (edgarordonez) [reply src/SingleBookingPage/NearestBooking.js]: Yes, agreed! How do you do this changes, in the same branch? Thanks @jalbertsr

Comment (jalbertsr) [reply src/SingleBookingPage/NearestBooking.js]: Yep, change it in the same branch 🙂

Comment (edgarordonez) [reply src/SingleBookingPage/NearestBooking.js]: Done! 🙂

**Pull request merged**